src/libraries/System.Resources.Reader/tests/ResourceReaderUnitTest.cs

```diff
@@ -259,8 +259,8 @@ public static void Exception_Enumerator_Value()
             });
         }
 
-        [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/51833", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
+        // Trimming, including NativeAot, will set CustomResourceTypesSupport to false
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBuiltWithAggressiveTrimming))]
```

Comment:
It might make sense to introduce yet another PlatformDetection constant for CustomResourceTypesSupport like Marek suggested in https://github.com/dotnet/runtime/issues/51833#issuecomment-873270937.